src/App.jsx

```diff
@@ -1,12 +1,31 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import Home from "./containers/home";
 import Login from "./containers/login";
+import { useSelector } from "react-redux";
+import { auth } from "database";
+import Profile from "containers/profile";
 
 function App() {
-  return (
+  console.log();
+  const [user, setUser] = useState({});
+
+  useEffect(() => {
+    setUser(userFromState);
+  }, []);
+
+  const userFromState = useSelector((state) => state.users.user);
+  return !userFromState.full_name ? (
```

Comment:
You can take what this means and put it in a variable to make it more readable. Something like `doesUserExist` or whatever `userFromState.full_name` captures

Reply:
done